### PR TITLE
Remove smart quotes

### PR DESCRIPTION
**Pull request by morganchen12**

### CODE REVIEW

File: Firebase/Database/Constants/FConstants.m [hunk 1]
```diff
@@ -41,7 +41,7 @@
 NSString *const kFWPASyncServerListenCancelled = @"c";
 NSString *const kFWPAsyncServerSecurityDebug = @"sd";
 NSString *const kFWPAsyncServerDataUpdateBodyPath =
-    @"p"; // {“a”: “d”, “b”: {“p”: “/”, “d”: “<data>”}}
+    @"p"; // {"a": "d", "b": {"p": "/", "d": "<data>"}}
```
Comment (paulb777) [inline Firebase/Database/Constants/FConstants.m]: @renkelvin - Are there missing tests for this one?

Comment (morganchen12) [reply Firebase/Database/Constants/FConstants.m]: This is in database. @schmidt-sebastian

Comment (paulb777) [reply Firebase/Database/Constants/FConstants.m]: Oh, I missed that. If it's not a recent change, I'm less worried - but still worth a review by @schmidt-sebastian

**Comment by renkelvin** [comment]
Looks good from Auth side

File: Firebase/Database/Constants/FConstants.m (hunk 1 above)
Comment (schmidt-sebastian) [reply Firebase/Database/Constants/FConstants.m]: LGTM. Looks like this is in the comment only.

**Pull request merged**